clusterloader2/pkg/measurement/common/probes/manifests/dnsPropagation/dns-propagation-prober-deployment.yaml

```diff
@@ -32,9 +32,12 @@ spec:
             - --logtostderr=true
             - --log_file=/var/log/cl2-dns-propagation.log
           resources:
+            requests:
+              cpu: {{.DNSPropagationProbeCpuRequest}}
+              memory: {{.DNSPropagationProbeMemoryRequest}}
             limits:
-              cpu: 100m
-              memory: 100Mi
+              cpu: {{.DNSPropagationProbeCpuRequest}}
```

Comment:
Supernit - maybe drop the `Request` suffix given we use that for `limits` as well?

Ditto for `DNSPropagationProbeMemoryRequest`.

Reply:
Fixed, thanks!